openapi/components/schemas/Subscription/OrderTypes/SubscriptionOrder.yaml

```
@@ -34,6 +34,17 @@ allOf:
         nullable: true
         type: integer
         readOnly: true
+      abandonReminderTime:
+        description: Time when the abandon order reminder event triggers.
+        nullable: true
+        type: string
+        format: date-time
+        readOnly: true
+      abandonReminderNumber:
+        description: Number of abandon order reminder events triggered.
```

nitpick: suggestion for readability 
```suggestion
        description: Number of abandon order reminder events that are triggered.
```